tests/unit/backend/corpora/api_server/test_v1_collection.py

```diff
@@ -603,6 +603,41 @@ def test__list_collection__check_owner(self):
             self.assertIn(private_owned, ids)
             self.assertNotIn(private_not_owned, ids)
 
+    def test__get_all_collections_for_index(self):
+        # dataset = self.generate_dataset(self.session, id="test_dataset_id_for_index", cell_count=42)
+        # self.generate_dataset(self.session, id="test_dataset_id_for_index_tombstone", tombstone=True)
+        # self.generate_dataset(self.session, id="test_dataset_id_for_index_private", collection_visibility="PRIVATE")
+
+        collection = self.generate_collection(
+            self.session, visibility=CollectionVisibility.PUBLIC.name, owner="test_user_id"
+        )
+
+        collection_id_private = self.generate_collection(
+            self.session, visibility=CollectionVisibility.PRIVATE.name, owner="test_user_id"
+        ).id
+
+        collection_id_tombstone = self.generate_collection(
+            self.session, visibility=CollectionVisibility.PUBLIC.name, tombstone=True, owner="test_user_id"
+        ).id
+
+        test_url = furl(path="/dp/v1/collections/index")
+        headers = {"host": "localhost", "Content-Type": "application/json", "Cookie": get_auth_token(self.app)}
```

Comment:
Is an auth token need for this endpoint?

Reply:
Nope, good call. Thanks!